src/olympia/reviewers/models.py

```diff
@@ -832,10 +834,12 @@ def get_base_addons(cls):
 
     def get_tier_boundaries(self):
         """Return boundaries used to filter add-ons for that tier instance."""
-        return {
-            'average_daily_users__gte': self.lower_adu_threshold,
-            'average_daily_users__lt': self.upper_adu_threshold,
-        }
+        # We always have a lower bound, but not necessarily a higher one, e.g.
+        # notable doesn't have a ceiling.
+        filters = {'average_daily_users__gte': self.lower_adu_threshold or 0}
+        if self.upper_adu_threshold is not None:
+            filters['average_daily_users__lt'] = self.upper_adu_threshold
```

Comment:
idea: would it make it easier to to some MAX_INT type value so it always exists?